src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
Arrg, it's why i don't try to innove...copy/paste is fine for me.
Now that i've changed, the test fail...

So i revert or i delete test...since i don't know if i could fix it. Let's see it tomorrow, better.